server/tserver/src/main/java/org/apache/accumulo/tserver/metrics/TabletServerUpdateMetrics.java

```diff
@@ -35,7 +36,7 @@ public class TabletServerUpdateMetrics implements MetricsProducer {
   private Timer commitPrepStat;
   private Timer walogWriteTimeStat;
   private Timer commitTimeStat;
-  private DistributionSummary mutationArraySizeStat;
+  private DistributionSummary mutationArraySizeStat = new NoOpDistributionSummary();;
```

Comment:
```suggestion
  private DistributionSummary mutationArraySizeStat = new NoOpDistributionSummary();
```
Unnecessary semi colon

Reply:
removed in 98e22de70e